### ISSUE TITLE
Path part of static export URL is ignored

### ISSUE THREAD

**Issue by raoulb** [pretalx/pretalx#1480]
### Problem and impact

Upon exporting a schedule for static web hosting, only the domain part of the `HTML-Export-Link` setting is accounted for. The path part is ignored.

This makes it impossible (well, at least hard) to host the files on a web server at some non top-level path w/o special server-side configuration. Generated internal links like `<link rel="icon" type="image/png" sizes="16x16" href="/static/common/img/favicon-16x16.02f8a56e1af3.png">` are absolute and just won't work.

This further impacts archiving of old schedules, for example hosting a static export at `https://example.org/schedule-archive/<year>/*` which I assume is a common use case for html export.

### Expected behaviour

Generate (internal) URLs within the exported files respecting the path part of the given configuration setting `HTML-Export-Link`.

### Version

v2.3.2

**Comment by rixx** [comment]
tl;dr: I just spent two hours looking into how we might fix this, and I'm sorry, but barring outside contributions (that I'd very much welcome – ideally in code, but if no other way can be found, then in sponsoring), this is out of scope for pretalx right now.

I found a way to adjust most-if-not-all rendered object links, since pretalx uses a shared base class for all event URLs. However – the way our export works, it absolutely breaks if we change URLs, since the export works by parsing exported pages for URLs (to make sure we get no dead links). But since the exported URLs already start with the new path, those 404, and the export gets stuck.

Additionally, to comply with this request, we'd also have to change at the very least how static and media urls are rendered (and these don't have an easy common root), and adjust links in CSS files (fonts and other assets).

As pretalx doesn't export an event to the actual site root (it does keep the event slug as prefix, so different event exports won't collide), I believe this is not urgent enough to prioritise given that we'd have to rework the entire export. Sorry to disappoint.

(Really sorry, but as just a single dev, my choice really is to get people to contribute code or funding, and close all issues that are so far outside my non-funded roadmap that they'll never get implemented, realistically. Like I said: happy to take a PR!)
